build.gradle

```diff
@@ -12,7 +12,7 @@ buildscript {
     }
     dependencies {
         classpath "io.sweers.redacted:redacted-compiler-gradle-plugin:$redacted_version"
-        classpath 'com.android.tools.build:gradle:4.0.0'
+        classpath 'com.android.tools.build:gradle:4.0.1'
```

Comment:
Let's revert this, we've done this internally to ensure the verification metadata is updated.

Reply:
do you mean the verication metadata? or the gradle version?

Reply:
Just the gradle version.

If you'd like to update the verification metadata to include the new version of Fastadapter, run `./gradlew --write-verification-metadata sha256,pgp --export-keys`.  We'll verify the metadata changes internally, but this will allow a build to succeed.

Reply:
fair enough, done!